src/ngraph/graph_util.cpp

```diff
@@ -530,18 +545,18 @@ size_t ngraph::get_user_count(Node* node)
 
 bool ngraph::possibly_overwritten(Node* node)
 {
-    for (const descriptor::Output& output : node->get_outputs())
+    for (auto& output : node->outputs())
```

Comment:
I think _technically_ no according to our coding standards (which I don't agree with on this point anyway), because `get_` is supposed to be used for trivial getters, which this is not (at least not yet). I kinda feel like it reads better as `outputs()`, but I wouldn't have major aesthetic objections to changing it.

For now the problem is that `get_outputs()`, which returns `deque<descriptor::Output*>`, already exists (albeit deprecated) and can't be overloaded.

Reply:
I agree with you that I disagree with the coding standard. The issue I have is that if I change the `implementation` of a method I may then need to change the `method name`. Exposing the implementation indirectly this way is not needed and not followed in our coding. The fact that this rule is often blatantly ignored means it is a bad rule and should be changed.